RadarSDK/RadarSettings.h

```diff
@@ -32,11 +32,15 @@ NS_ASSUME_NONNULL_BEGIN
 + (void)setAdIdEnabled:(BOOL)enabled;
 + (BOOL)tracking;
 + (void)setTracking:(BOOL)tracking;
++ (BOOL)listenToServerTrackingOptions;
```

Comment:
From what I can see, this setting is purely for logging, is that right @jkao ?  Doesn't affect behavior except to pass up a req.param with its value.

cc @jsani-radar @jrtibbetts

Reply:
seems like it, to measure on the BE how many clients are using server-side options

the var name makes it sound more like a setter / command / action than an internal value - would something like `listeningToServerTrackingOptions` be better?

Reply:
`listeningToServerTrackingOptions` sounds ok to me. `listenToServerTrackingOptions` would sound like a command. Maybe `isListening...` or `shouldListenTo...` would be better? Or maybe not `listen`, but `use`?

Reply:
yeah `listening` is probably a better tense, but you are correct in the sense that it is for logging. you can see this in Redshift and Athena locations